lib/services/access_request.go

```diff
@@ -1690,17 +1690,29 @@ func GetResourceDetails(ctx context.Context, clusterName string, lister Resource
 
 	result := make(map[string]types.ResourceDetails)
 	for _, resource := range resources {
-		hn, ok := resource.(interface{ GetHostname() string })
-		if !ok {
+		// Right now, only Okta resources have friendly names.
+		var friendlyName string
+		if resource.Origin() == types.OriginOkta {
+			friendlyName = resource.GetMetadata().Description
+		}
+
+		var hostname string
+		if hn, ok := resource.(interface{ GetHostname() string }); ok {
+			hostname = hn.GetHostname()
+		}
```

Comment:
is it correct that this block of code should display my access requested nodes as `node: <hostname>`?? i'm still seeing `node: <uuid>` but i might be misunderstanding

Reply:
The node:uuid shows in the "view all access requests" table. We don't pull hostnames there (we should, but it was a TODO on a PR from awhile ago that nik added. I'll have to find it to remember why). This change is for the individual request review page (where you approve or deny), where the resources requested are listed in that table.